frontend/www/js/omegaup/components/common/Navbarv2.vue

```diff
@@ -415,6 +430,22 @@ export default class Navbar extends Vue {
   readNotifications(notifications: types.Notification[], url?: string): void {
     this.$emit('read-notifications', notifications, url);
   }
+
+  @Watch('toggleSwitch')
+  onSwitchChanged(newValue: boolean) {
+    const isDark = !newValue;
+    if (isDark) {
+      document
+        .getElementById('dark-theme-style')
+        ?.setAttribute(
+          'href',
+          'https://bootswatch.com/4/cyborg/bootstrap.min.css',
+        );
+    } else {
+      document.getElementById('dark-theme-style')?.setAttribute('href', '');
+    }
+    localStorage.setItem('toggle-bootstrap-theme', JSON.stringify({ isDark }));
```

Comment:
aquí sólo habría que hacer

```suggestion
    localStorage.setItem('theme-preferences', JSON.stringify({ isDark }));
    ui.updateTheme();
```

Reply:
Listo